external-docs/docs/guides/shape-conversion.md

```diff
@@ -0,0 +1,99 @@
+# Converting data shape
```

Comment:
Can we link to this from another document? Perhaps all 3 of the shaping data docs?

Reply:
Done